src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaTestHelper.java

```diff
@@ -110,4 +114,14 @@ public static void cleanUpLogs() {
       e.printStackTrace();
     }
   }
+
+  public static void truncate(File file) {
+    try (FileChannel outChan = new FileOutputStream(file, false).getChannel()) {
+      outChan.truncate(0);
+    } catch (FileNotFoundException e) {
+      System.out.println(file.getName() + " does not exist.");
```

Comment:
Sysout or log?

Reply:
For tests System.out is okay. We don't usually look at logs.